onnxruntime/core/providers/coreml/builders/impl/reshape_op_builder.cc

```diff
@@ -94,6 +96,18 @@ bool ReshapeOpBuilder::IsOpSupportedImpl(const InitializedTensorSet& initializer
     return false;
   }
 
+  // CoreML reshape does not support 0 as dimension
+  NodeAttrHelper helper(node);
+  bool allow_zero = helper.Get("allowzero ", 0) == 1;
+  if (allow_zero) {
+    for (int64_t i = 0; i < perm_dims[0]; i++) {
+      if (raw_shape[i] == 0) {
+        LOGS_DEFAULT(VERBOSE) << "Reshape doesn't suppport 0 reshape dimension when allowzero is enabled";
```

Comment:
>suppport [](start = 50, length = 8)

"suppport" -> "support"

Reply:
also shows up in op_support_checker.cc

---
In reply to: [606023324](https://github.com/microsoft/onnxruntime/pull/7211#discussion_r606023324) [](ancestors = 606023324)